### DISCUSSION TITLE
Help a newbie

### DISCUSSION THREAD

**Discussion by drgshah** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#70] [Q&A] [answered]
I am sorry if this is a noob question.
How do i insert the configuration (where?)
the readme gives a config example, but where do i insert it?

I am certainly missing the obvoious, since others seem to know what to do.
Thanks

**Comment by PiotrMachowski** [comment, marked as answer]
You have to configure it in [configuration.yaml](https://www.home-assistant.io/docs/configuration/) file

**Reply by drgshah** [reply]
Thanks. Got it to work.

**Reply by PiotrMachowski** [reply]
Great 👍
